cmd/flags.go

```diff
@@ -24,7 +24,12 @@ var (
 	DefaultProcessType         = "web"
 	DefaultProjectMetadataPath = filepath.Join(".", "project-metadata.toml")
 	DefaultReportPath          = filepath.Join(".", "report.toml")
+	DefaultStackOrderPath      = filepath.Join(rootDir, "cnb", "stack", "order.toml")
```

Comment:
The RFC has this path as `/cnb/stack/buildpacks/order.toml`, I believe. Not sure if the RFC or the PR should be updated, but probably it's worth keeping things consistent.

Reply:
@jkutner thoughts?